daiquiri/tests/test_daiquiri.py

```diff
@@ -94,6 +94,11 @@ def test_string_as_setup_outputs_arg(self):
         if daiquiri.handlers.journal is not None:
             daiquiri.setup(outputs=("journal",))
 
+    def test_special_kwargs(self):
+        daiquiri.getLogger(__name__).info(
+            "foobar", foo="bar", exc_info=True, stack_info=True
+        )
```

Comment:
Hum I'm not sure this tests your change. This worked before, no?

Reply:
Oh I realize you had an exception without that sorry. I thought it was passed down!

Reply:
nope it didn't! It should test exactly my change (and also the "exc_info" which was handled correctly before). Or am I missing something?

Reply:
Ok! Missed your second comment